src/install/python/python-options/python-agent-uvicorn-docker.mdx

```diff
@@ -0,0 +1,113 @@
+---
+componentType: appInfo
+optionType: docker
+headingText: Set up Uvicorn in Docker
+---
+
+Here are some instructions for doing automatic initialization with Uvicorn and `docker run`. If you are using another supported framework with Docker, you'll need to adjust your syntax.
+
+### A. Create the directory structure [#create-directories]
+
+You'll need to insert some files in the directory structure for your application. To prepare for that, you might create a structure similar to this:
+
+```
+python-docker
+├── newrelic
+│   └── Dockerfile
+└── src
+    ├── Dockerfile
+    ├── your_app.py
+    ├── newrelic.ini
+    └── requirements.txt
+```
+
+### B. Insert the New Relic configuration file [#config-file]
+
+In the previous steps, you created a `newrelic.ini` configuration file that contains your license key and app name.
+
+Insert this configuration in the same directory as your Python app.
+
+### C. Create the requirements file [#requirements-file]
+
+Create `requirements.txt` (the list of dependencies for Python) by running this command in the directory where your application resides:
+
+```
+pip freeze > requirements.txt
+```
+
+### D. Create the base container (New Relic) [#base-container]
+
+This is the container that runs the agent.
+
+1. In the directory for the New Relic agent (for example, `newrelic`), create this Dockerfile for the base container (the file name is just `Dockerfile`):
+
+    ```
+    FROM python:3.9.14-alpine3.16
+
+    RUN pip install --no-cache-dir newrelic
+
+    ENTRYPOINT ["newrelic-admin", "run-program"]
+    ```
+
+2. Create the container with this command:
+
+    ```
+    docker build -t python_newrelic:latest .
+    ```
+
+3. Change to the directory where your app is located (for example, src)
+
+### E. Create a container for your app
+
+This is the container that will hold your application. It will work in concert with the base container. Note that it pulls in the base container in the `FROM`.
+
+1. Create the Dockerfile for your app in the same directory as your Python app.
+
+    * Change `WORKDIR` to match your directory structure.
+    * For the `CMD` line, replace the uvicorn example with the syntax from the previous step.
```

Comment:
I'm having trouble understanding what the user is supposed to replace the CMD line with. Could we add some clarification around that here? Rather than pointing to the previous step maybe we can just say what to do directly here?